data_safe_haven/infrastructure/programs/sre/dns_server.py

```diff
@@ -69,6 +72,9 @@ def __init__(
         )
 
         # Expand AdGuardHome YAML configuration
+        mustache_values: dict[str, object] = {
+            "allow_workspace_internet": props.allow_workspace_internet
+        }
```

Comment:
This feels a bit redundant, can we put this in the `Output.all` statement below?